### PR TITLE
Go v1.19

### PR DESCRIPTION
**Pull request by kon-angelo**
**How to categorize this PR?**

/area control-plane
/kind enhancement
/priority normal

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Comment by kon-angelo** [comment]
/hold

I need to test new images before proceeding

**Comment by kon-angelo** [comment]
The PR should be ready for review. I will update the golang version and will unhold

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ############# golang-base
-FROM golang:1.16.15 AS golang-base
+FROM golang:1.19.3 AS golang-base
```
Comment (MartinWeindel) [inline Dockerfile:6]: Let's use the latest 1.19 version
```suggestion
FROM golang:1.19.5 AS golang-base
```

**Review by MartinWeindel — COMMENTED** [review]
You may want to use 1.19.5 in the meantime

**Review by MartinWeindel — APPROVED** [review]
/lgtm

**Pull request merged**